integrations/repo_search_test.go

```diff
@@ -27,6 +31,23 @@ func resultFilenames(t testing.TB, doc *HTMLDoc) []string {
 func TestSearchRepo(t *testing.T) {
 	prepareTestEnv(t)
 
+	repo, err := models.GetRepositoryByOwnerAndName("user2", "repo1")
+	assert.NoError(t, err)
+
+	models.UpdateRepoIndexer(repo)
+
+	log.Printf("Waiting for indexing\n")
+
+	i := 0
+	for {
```

Comment:
```suggestion
	for i < 60 {
	if (repo.IndexerStatus != nil && len(repo.IndexerStatus.CommitSha) != 0) {
```
Might be a bit more readable, but thats just a small nit.